core/unit_test/TestTeamBasic.hpp

```diff
@@ -182,6 +182,7 @@ struct LargeTeamScratchFunctor {
 };
 
 TEST(TEST_CATEGORY, large_team_scratch_size) {
+#if !KOKKOS_IMPL_32BIT
```

Comment:
See https://github.com/kokkos/kokkos/pull/5916#discussion_r1161881068
```suggestion
#ifdef KOKKOS_IMPL_32BIT
  GTEST_SKIP() << "Fails on 32-bit";  // FIXME_32BIT
#endif
```